neuralmonkey/decoders/beam_search_decoder.py

```diff
@@ -181,9 +191,22 @@ def cond(*args) -> tf.Tensor:
         dec_loop_state = final_state.decoder_loop_state
         bs_state = final_state.bs_state
 
-        scores = final_state.bs_output.scores.stack()
-        parent_ids = final_state.bs_output.parent_ids.stack()
-        token_ids = final_state.bs_output.token_ids.stack()
+        # Because the "batch" dymension is dynamic and the TensorArrays
```

Comment:
Man schreibt: "dimension"

Reply:
hahaha, dylina

Reply:
me se to hrozne plete s dynamitem

Reply:
Pročs to neopravil, dynamite?

Reply:
Ale stejne, kdyz je dYnamicka, tak by to mohla byt dYmenze, ne? :)